### DISCUSSION TITLE
阿里云盘You are denied by bucket referer policy

### DISCUSSION THREAD

**Discussion by zhujitop** [AlistGo/alist#1044] [Q&A]
阿里云盘出现：
You are denied by bucket referer policy.

#17 
看了这个，我也把：
`<meta name="referrer" content="never">`

加到网盘程序头部还是会出现这种被拒绝的情况。
如果打开web proxy之后，能正常一段时间，一段时间又不行了，得切换成默认方式下载。
就是两种方式来回切换，是操作方式不对吗？

**Comment by yizhimengxin2233** [comment]
所以解决了吗？

**Comment by woshichenghaibo** [comment]
我从来没解决这个问题，文件头加上也没用

**Reply by anwen-anyi** [reply]
浏览器问题 换个就行

**Comment by woshichenghaibo** [comment]
我看到的是有的人加的文件，头不是这么写的，是这么写<meta name="referrer" content="no-referrer">
